File: JavaScriptBasics/WhileLoop/Exercise/OldBooks.js
function OldBooks(input){

    let book = input[0];

    let index = 1;

    let check = input[index];

    let found = false;

    while(book!==check || check=="No More Books"){
index++;
check=input[index];
if(check==="No More Books"){break;}
if(book===check){
    found=true;
}
    }
if(found){
    console.log("You checked "+(index-1)+" books and found it.");
}else{
    console.log("The book you search is not here!");
    console.log("You checked "+(index-1)+" books.");
}
}

File: JavaBasics/WhileLoops/Exercise/OldBooks.java
package EntryModule.WhileLoops.Exercise;

import java.util.Scanner;
public class OldBooks {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String favourite = scanner.nextLine();
        String firsttry = scanner.nextLine();
        int times = 0;
        if(favourite.equals(firsttry)){
            System.out.println("You checked 0 books and found it.");
        }

        while(!favourite.equals(firsttry)){
            firsttry = scanner.nextLine();
            times++;
            if(firsttry.equals("No More Books")){
                System.out.println("The book you search is not here!");
                System.out.println("You checked "+times+" books.");
                break;
            }
            if(firsttry.equals(favourite)){
                System.out.println("You checked "+times+" books and found it.");
                break;
            }
        }
    }
}
